Traceback (most recent call last):
  File "D:\cedri\anaconda3\envs\prepa\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "D:\cedri\anaconda3\envs\prepa\lib\site-packages\nbclient\client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "D:\cedri\anaconda3\envs\prepa\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "D:\cedri\anaconda3\envs\prepa\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "D:\cedri\anaconda3\envs\prepa\lib\asyncio\base_events.py", line 587, in run_until_complete
    return future.result()
  File "D:\cedri\anaconda3\envs\prepa\lib\site-packages\nbclient\client.py", line 560, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "D:\cedri\anaconda3\envs\prepa\lib\site-packages\nbclient\client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "D:\cedri\anaconda3\envs\prepa\lib\site-packages\nbclient\client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from reponses.reponses import gaussienne
m = 2  # Esp√©rance
s = 0.3  # Ecart-type
f = gaussienne(m, s)
plt.show()
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mNameError[0m                                 Traceback (most recent call last)
[1;32m~\AppData\Local\Temp/ipykernel_3120/1560802341.py[0m in [0;36m<module>[1;34m[0m
[0;32m      3[0m [0ms[0m [1;33m=[0m [1;36m0.3[0m  [1;31m# Ecart-type[0m[1;33m[0m[1;33m[0m[0m
[0;32m      4[0m [0mf[0m [1;33m=[0m [0mgaussienne[0m[1;33m([0m[0mm[0m[1;33m,[0m [0ms[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 5[1;33m [0mplt[0m[1;33m.[0m[0mshow[0m[1;33m([0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m
[1;31mNameError[0m: name 'plt' is not defined
NameError: name 'plt' is not defined

